changelog/struct_ctor_default_params.md

```diff
@@ -0,0 +1,10 @@
+Struct constructors with all-default parameter will now error
+
+Since 2.070.0, the following code has been giving deprecations:
+```
```

Comment:
This isn't markdown. It's Ddoc. Also mind the extension.

Reply:
Good catch! I did that mistake in #9754 as well, so adding a fix for that.